src/main/java/org/jabref/logic/importer/AuthorListParser.java

```diff
@@ -209,6 +180,100 @@ public AuthorList parse(@NonNull String listOfNames) {
         return AuthorList.of(authors);
     }
 
+    /**
+     * Attempts to parse a single author string using the BibLaTeX extended name format.
+     * The format includes attributes such as family, given, prefix, and suffix.
+     * Also handles the 'useprefix=false' case where the prefix should be ignored.
+     *
+     * @param authorString the string representing a single author in extended format
+     * @return Optional containing an Author object if parsing is successful, or empty if not
+     */
+    private Optional<Author> parseExtendedNameFormat(String authorString) {
+        Map<String, String> nameParts = new HashMap<>();
+        Matcher matcher = Pattern.compile("(\\w+)\\s*=\\s*([^,]+)(?:,\\s*|$)").matcher(authorString);
```

Comment:
Please compile the regex only once. Compiling regular expressions is an expensive operation.